File: Const.cs
namespace MegaMini
{
    internal class Const
    {
        public const int bufferSize = 1024 * 64;        
        public const int responseTimeout = Timeout.Infinite;
        public const string applicationKey = "axhQiYyQ";
        public const string baseLink = "https://g.api.mega.co.nz/cs";
        public static uint sequenceIndex = (uint)(uint.MaxValue * new Random().NextDouble());
    }
}

File: android/app/src/main/java/com/megamini/android/Const.kt
package com.megamini.android

/**
 * Константы для Mega.nz API
 */
object Const {
    const val bufferSize: Int = 1024 * 64
    const val responseTimeout: Int = -1 // Бесконечный таймаут
    const val applicationKey: String = "axhQiYyQ"
    const val baseLink: String = "https://g.api.mega.co.nz/cs"
    var sequenceIndex: UInt = (UInt.MAX_VALUE * Math.random()).toUInt()
}
